packages/studio-ui/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I saw that in the `tsconfig` of studio-plugin, we extend the config from the root `tsconfig` since a lot of it is the same. Is that something that we would want to do for all the `tsconfig`s?  I'm also not entirely clear how this works outside of `studio-prototype` in npm world.